src/MainWindow.cpp

```diff
@@ -1200,6 +1207,7 @@ void MainWindow::executeQuery()
     // NOTE It's a bit hack-ish but we don't use this icon just as a signal to the user but also check for it in various places to check whether a
     // specific SQL tab is currently running a query or not.
```

Comment:
Can you remove that comment, not that it is outdated? Or better yet, update it to explain that the tab data value means :smile:

Reply:
Of course. I'll update.